deploy/dev/5_load_environment.sh

```diff
@@ -24,9 +35,20 @@ function configure_secret() {
   echo "Create secret k8s-secret"
   $cli_with_timeout create -f $CONFIG_DIR/k8s-secret.yml
 
+  create_k8s_resources
+}
+
+function create_k8s_resources {
   create_secret_access_role
 
   create_secret_access_role_binding
 }
 
+function deploy_chart {
```

Comment:
this function is implemented also in `reload.sh`. is this on purpose?

Reply:
moved to utils (: